var/spack/repos/builtin/packages/bmake/package.py

```diff
@@ -6,29 +6,42 @@
 from spack import *
 
 
-class Bmake(Package):
+class Bmake(AutotoolsPackage):
```

Comment:
The reason that this originally used `Package` instead of `AutotoolsPackage` is that by bootstrapping the program, we can build `make` without relying on a previously existing `make` on the system. I'm not sure if anyone has actually used it for that purpose, but this change makes it impossible to bootstrap `make` without an external `make` command.

Reply:
Got it.